client/src/app/shared/instance/instance-features-table.component.scss

```diff
@@ -27,6 +28,13 @@ table {
   td {
     vertical-align: middle;
   }
+
+  caption {
+    caption-side: top;
+    font-size: 16px;
+    font-weight: $font-semibold;
```

Comment:
```suggestion
    font-size: 15px;
    font-weight: $font-bold;
```

Reply:
This is to provide the same style as `label` elements, but you might find it too bold - which do you prefer?

Reply:
I find the $font-bold too bold 😄 
I'll change the font-size :)